server/src/main/java/com/showmetheroapes/paracord/controllers/StrandController.java

```diff
@@ -0,0 +1,24 @@
+package com.showmetheroapes.paracord.controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/api/v1/strands", produces = "application/json")
```

Comment:
I wonder if the produces automatically sticks the Content-Type in the headers...

Reply:
Correct! Its telling spring how to serialize the returned objects

Reply:
Noice!